server/server.js

```diff
@@ -4,41 +4,18 @@ var boot = require('loopback-boot');
 
 var app = module.exports = loopback();
 
-// middleware
-app.use(loopback.compress());
-
-// it's important to register the livereload middleware
-// after any response-processing middleware like compress,
-// but before any middleware serving actual content
-var livereload = app.get('livereload');
-if (livereload) {
-  app.use(require('connect-livereload')({
-    port: livereload
-  }));
-}
-
 // boot scripts mount components like REST API
 boot(app, __dirname);
 
-// Mount static files like ngapp
-// All static middleware should be registered at the end, as all requests
-// passing the static middleware are hitting the file system
-app.use(loopback.static(path.dirname(app.get('indexFile'))));
-
-// Requests that get this far won't be handled
-// by any middleware. Convert them into a 404 error
-// that will be handled later down the chain.
-app.use(loopback.urlNotFound());
-
-// The ultimate error handler.
-app.use(loopback.errorHandler());
-
 // optionally start the app
 app.start = function() {
   // start the web server
   return app.listen(function() {
     app.emit('started');
-    console.log('Web server listening at: %s', app.get('url'));
+    var baseUrl = app.get('url').replace(/\/$/, '');
+    var explorerPath = app.get('loopback-explorer').mountPath;
```

Comment:
This relies on loopback-explorer being loaded via component-config.json. When the user decides to disable explorer (e.g. in production via component-config.production.json), the app will crash.

Please rework this line to check if `app.get('loopback-explorer')` returns a valid value and don't print the message "Browser your REST API" otherwise.